elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/query/QueryTranslator.java

```diff
@@ -282,6 +282,7 @@ private String extractOrderBy(Map<Path, Sorting.SortOrder> sortClauses, Query qu
      */
     private Set<String> extractJoinExpressions(Query query) {
         return query.getColumnProjections().stream()
+                .filter(column -> column.isProjected())
```

Comment:
The params above in the javadoc are wrong.  This should explain that this only extracts joins from projected columns.